proposals/description/fip_99.ts

```diff
@@ -21,22 +21,30 @@ const fip_99: ProposalDescription = {
       target: 'core',
       values: '0',
       method: 'grantMinter(address)',
-      arguments: ['{globalRateLimitedMinter}'],
-      description: 'Grant the MINTER_ROLE to the global rate limited minter'
+      arguments: ['{raiPriceBoundPSM}'],
+      description: 'Grant the MINTER_ROLE to the raiPriceBoundPSM'
     },
     {
       target: 'core',
       values: '0',
       method: 'grantPCVController(address)',
-      arguments: ['{raiNonCustodialPriceBoundPSM}'],
-      description: 'Grant the PCV_CONTROLLER_ROLE to the non-custodial price-bound psm'
+      arguments: ['{raiPCVDripController}'],
+      description: 'Grant the PCV_CONTROLLER_ROLE to the rai pcv drip controller'
     },
     {
-      target: 'raiNonCustodialPriceBoundPSM',
+      target: 'raiPriceBoundPSM',
       values: '0',
-      method: 'pauseRedeem()',
+      method: 'pauseMint()',
       arguments: [],
-      description: 'Pause redeem on the non-custodial price-bound psm'
+      description: 'Pause mint on the rai price-bound psm'
+    },
+    {
+      target: 'pcvGuardian',
+      values: '0',
+      method: 'setSafeAddresses(address[])',
+      arguments: [['{rariPool9RaiPCVDeposit}', '{aaveRaiPCVDeposit}', '{raiPriceBoundPSM}']],
+      description:
+        'Whitelist the fuse pool 9 rai pcv deposit and the aave rai pcv deposit on the pcv guardian, as well as the new rai psm'
     }
```

Comment:
Honestly this may be the only action actually needed. The first two can be done by the PCVGuardian, the minter isn't needed. Movements can be done by the PCVGuardian too. Minting doesn't need to be paused if the contract doesn't have minter.

If we want to keep PCV Dripper thats fine though

Reply:
PCVGuardian will require a manual transaction to "drip" every time, and especially if we're using smaller-size drips, that doesn't seem very feasible compared to an actual pcv drip controller.

Reply:
Drips should be greedy and as large as possible right?

Reply:
Going with drip size of 2.5m for now